src/table_builder.cc

```diff
@@ -97,6 +97,7 @@ void TitanTableBuilder::Add(const Slice& key, const Slice& value) {
       // If not ok, write original blob index as compaction output without
       // doing level merge.
       if (get_status.ok()) {
+        level_merge_performed_ = true;
```

Comment:
I think `TITAN_GC_NUM_KEYS_RELOCATED` is better, and should count the blobs written by `blob_run_mode` as well.